packages/cli/lib/utils.js

```diff
@@ -370,3 +370,24 @@ function validateValue(name, unallowedCharacters) {
   }
   return true;
 }
+/**
+ *  Returns the modelName in the directory file format for the model
+ * @param {string} modelName
+ */
+exports.toModelFileName = function(modelName) {
+  return `${_.kebabCase(modelName)}.model.ts`;
+};
+
+/**
+ * Returns the repositoryName in the directory file format for the repository
+ * @param {string} repositoryName
+ */
+exports.toRepositoryFileName = function(repositoryName) {
```

Comment:
Perhaps `getRepositoryFileName` would be a better function name?

Reply:
@bajtos , I thought about this name, however, I wanted to keep consistency with the other method toClassName in the same utils.js file. so we have toClassName, toRepositoryFileName etc.

Reply:
I see. In that case I am ok with either way.